servers/camera/camera_feed.h

```diff
@@ -36,6 +36,8 @@
 #include "servers/camera_server.h"
 #include "servers/rendering_server.h"
 
+#include <vector>
```

Comment:
It doesn't look like this is used? In general, we don't use `std::vector` much in Godot.